addons/advanced_ballistics/XEH_postInit.sqf

```diff
@@ -22,3 +22,17 @@ if (!GVAR(extensionAvailable)) exitWith {
 };
 */
 [] call FUNC(initializeTerrainExtension);
+
+if (!hasInterface) exitWith {};
+
+["SettingsInitialized", {
+    //If not enabled, dont't add PFEH
+    if (!GVAR(enabled)) exitWith {};
+
+    // Register fire event handler
+    ["firedPlayer", DFUNC(handleFired)] call EFUNC(common,addEventHandler);
+    ["firedPlayerNonLocal", DFUNC(handleFired)] call EFUNC(common,addEventHandler);
```

Comment:
i think here we can add a what we check in  addons/advanced_ballistics/functions/fnc_handleFired.sqf Line 28